kahuna/app/KahunaComponents.scala

```diff
@@ -35,7 +35,7 @@ object KahunaSecurityConfig {
     )
 
     val frameSources = s"frame-src ${config.services.authBaseUri} ${config.services.kahunaBaseUri} https://accounts.google.com"
-    val frameAncestors = s"frame-ancestors ${config.services.toolsDomains.map(domain => s"*.$domain").mkString(" ")}"
+    val frameAncestors = s"frame-ancestors ${config.services.frameAncestors.mkString(" ")}"
```

Comment:
Just for my own understanding -- is the renaming here just a matter of clarifying what this config param is used for?

Reply:
Yes, you're correct;`toolsDomains` is very Guardian-y.